docs/topics/proc_deploying-a-booster-to-openshiftcontainerplatform.adoc

```diff
@@ -29,26 +30,26 @@ $ oc login OCP_URL --token=MYTOKEN
 $ oc new-project {value-name-project}
 ----
 
-. Navigate to the root directory of your booster.
+. Navigate to the root directory of your example.
 
-. Deploy your booster to {OpenShiftContainerPlatform}.
-.. If your booster uses Maven: 
+. Deploy your example to {OpenShiftContainerPlatform}.
+.. If your example uses Maven: 
 +
 [source,bash,options="nowrap",subs="attributes+"]
 ----
 $ mvn clean fabric8:deploy -Popenshift
 ----
-.. If your booster uses `npm`.
+.. If your example uses `npm`.
 +
 [source,bash,options="nowrap",subs="attributes+"]
 ----
 $ npm install && npm run openshift
 ----
 
-. Open the `README.adoc` file in your booster's project and follow the instructions for interacting with your booster.
+. Open the `README.adoc` file in your example's project and follow the instructions for interacting with your example application.
 
 .Additional resources
-Take a look at the different runtime guides to learn more about runtimes as well as their boosters:
+Take a look at the different runtime guides to learn more about runtimes as well as their examples:
```

Comment:
This sentence seems semantically weird to me, and does not conform to the glossary definitions.

Reply:
I changed the wording, please let me know if it is better.